packages/opencensus-core/src/internal/util.ts

```diff
@@ -33,3 +33,42 @@ const spanRandomBuffer = randomFillSync ?
 export function randomSpanId() {
   return spanRandomBuffer().toString('hex');
 }
+
+/** Common methods to encode and decode varints and varlongs into ByteBuffers and arrays. */
+export class VarInt {
+  
+  static varIntSize(i: number): number {
```

Comment:
Nit: what would you think about making these top-level exported functions rather static members of a `VarInt` class (unless `VarInt` will contain state, in which case they could be member functions)?

Reply:
We are making use of bunch of methods like getVarInt, putVarInt,varIntSize so I felt housing them under a class would be the way to go.

Reply:
OK. What would you think about using a module for that? There are some static classes used as containers in the library, but I tend to agree with this [Stack Overflow answer](https://stackoverflow.com/a/32984233) that TS modules are a nice way to do that type of grouping vs. static classes. Modules have similar encapsulation and naming (through the file) that a class has but also make it easier to reference (you can destructure if you want).

What do you think?

If you feel strongly about the static methods, I'm open assuming @mayurkale22 is OK.

Reply:
tslint gives the following error when trying to use modules.
`'namespace' and 'module' are disallowed `

Reply:
Oh, I should have specified, I meant just an ES6 module that is created by just making a new `.ts` file like `varint-util.ts` or similar. Could that work?

Reply:
My first thought was thought, but in the SO question namespace was mentioned, so I assumed that you specifically wanted that. my bad!